tests/macsec/test_interop_protocol.py

```diff
@@ -132,11 +133,33 @@ def test_snmp(self, duthost, ctrl_links, upstream_links, creds, wait_mka_establi
                     commands=['show snmp community | include name'])
                 community = re.search(r'Community name: (\S+)',
                                       result['stdout'][0]).groups()[0]
+                up_link = upstream_links[ctrl_port]
+                nbr_ip = up_link["local_ipv4_addr"]
             else:  # vsonic neighbour
                 community = creds['snmp_rocommunity']
+                # Use LoopbackIP to query from vsonic neighbor
```

Comment:
In the code comment, could you be more specific about ipv4 or ipv6?

Reply:
Fixed